tests/helpers/test_entity_registry.py

```diff
@@ -384,8 +410,9 @@ async def test_migration(hass):
     assert entry.config_entry_id == "test-config-id"
 
 
+@pytest.mark.parametrize("load_registries", [False])
 async def test_loading_invalid_entity_id(hass, hass_storage):
-    """Test we autofix invalid entity IDs."""
+    """Test we skip entities with invalid entity IDs."""
```

Comment:
I'm not quite sure what's going on here.
There's no autofixing of bad entity IDs, we just skip them when loading the entity registry.

Reply:
we used to have this logic as there was an edge case where we could have entities with `__` because of a bad slugify implementation.

Reply:
I think that we can drop this test now.